UITable/UITable/Controller/DetailController.swift

```diff
@@ -17,24 +17,17 @@ class DetailController: UIViewController {
     @IBOutlet weak var constrainBottomTopToView: NSLayoutConstraint!
     @IBOutlet weak var scrollView: UIScrollView!
     @IBOutlet weak var openWikiButton: UIButton!
+    @IBOutlet weak var visualButton: UIButton!
     
     let highPriority = Float(999)
     var isTextOpen = false
-    var model: EntityProtocol?
+    var model: EntityProtocol!
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        textLabel.text = model?.getDescr()
-        self.title = model?.getName()
-        viewForOpacity.opacityGradient()
-    }
-
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(true)
-    }
-    
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
+        textLabel.text = model.descr
```

Comment:
Don't shorten words in properties names (until it's really required).
`description` is` NSObject` property, so think about better name.